### DISCUSSION TITLE
Making the OIDC provider the default login method

### DISCUSSION THREAD

**Discussion by Node815** [nextcloud/all-in-one#4076] [Questions] [answered]
I have AIO protected by my SSO solution, (Authentik), they suggest to use this command to disable it, but it is not found while exec'd into the container which is really no surprise to me given the nature of AIO.  So, is there an equivalent to doing this? 

```sudo -u www-data php var/www/nextcloud/occ config:app:set --value=0 user_oidc allow_multiple_user_backends```

Their instructions can be found [here](https://goauthentik.io/integrations/services/nextcloud/)  for reference. 

I understand the thought process though if it's not supposed to be disabled for reasons of emergency recovery, but what if I really want to do this and accept all responsibility? :)

**Comment by szaimen** [comment, marked as answer]
Hi, based on https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-run-occ-commands it should be something like `sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ config:app:set --value=0 user_oidc allow_multiple_user_backends`

**Comment by sa-schiefer** [comment]
Be careful and don't rely on it as a security measure assuming that local logins won't be possible anymore as this can be overridden with an URL parameter.

nexcloud.tld/login?direct=1 will present the normal login page and expose accounts with likely outdated local passwords and non-existing local 2fa to attacks.

That's a huge security issue in my opinion, especially since it's not well known and can be easily overlooked.

I currently filter this parameter on my reverse proxy as a workaround but I don't want to know how many people and companies don't even know about it.
